vic/drivers/image/src/vic_init_output.c

```diff
@@ -160,12 +174,12 @@ initialize_history_file(nc_file_struct *nc)
         log_err("Error defining nlayer dimenension in %s", nc->fname);
     }
 
-    status = nc_def_dim(nc->nc_id, "ni", nc->ni_size, &(nc->ni_dimid));
+    status = nc_def_dim(nc->nc_id, "lon", nc->ni_size, &(nc->ni_dimid));
     if (status != NC_NOERR) {
         log_err("Error defining ni dimenension in %s", nc->fname);
     }
 
-    status = nc_def_dim(nc->nc_id, "nj", nc->nj_size, &(nc->nj_dimid));
+    status = nc_def_dim(nc->nc_id, "lat", nc->nj_size, &(nc->nj_dimid));
```

Comment:
Suggestion: Let the user choose from the global_param file. Shall we implement this?